app/src/main/java/fr/free/nrw/commons/review/ReviewActivity.java

```diff
@@ -91,6 +92,9 @@ protected void onCreate(Bundle savedInstanceState) {
         ButterKnife.bind(this);
         initDrawer();
 
+        yesButton=findViewById(R.id.yesButton);
+        noButton=findViewById(R.id.noButton);
+
```

Comment:
You can use @BindView for referencing variables, as we did for other views. Please remove findViewById methods to preserve consistency in the code.